include/BMS/dev/BQ76952.hpp

```diff
@@ -159,9 +159,18 @@ class BQ76952 {
      * @param[out] cellVoltages The buffer to fill with the cell voltage. Must be
      *            NUM_CELLS in size
      * @param[out] sum The total voltage across all cells
+     *
+     * @param[out] minVoltage Voltage value of the lowest cell voltage of all cells in the pack.
+     *
+     * @param[out] maxVoltage Voltage value of the highes cell voltage of all cells in the pack.
+     *
+     * @param[out] minCellVoltageID The ID of the cell with the lowest voltage.
+     *
+     * @param[out] maxCellVoltageId The ID of the cell with the highest voltage.
+     *
      * @return The status of the read attempt
      */
-    Status getCellVoltage(uint16_t cellVoltages[NUM_CELLS], uint32_t* sum);
+    Status getCellVoltage(uint16_t cellVoltages[NUM_CELLS], uint32_t* sum, int16_t* minVoltage, uint8_t* minCellVoltageID, int16_t* maxVoltage, uint8_t* maxCellVoltageId);
```

Comment:
NIT: May want to make a struct to hold voltage reading parameters to wrap up all the data and reduce the number of parameters